components/menu/test/menu-item-mixin.test.js

```diff
@@ -1,76 +1,80 @@
-import { defineCE, expect, fixture, oneEvent } from '@brightspace-ui/testing';
+import { clickElem, defineCE, expect, fixture, oneEvent, sendKeysElem } from '@brightspace-ui/testing';
+import { createMessage } from '../../../mixins/property-required/property-required-mixin.js';
 import { LitElement } from 'lit';
 import { MenuItemMixin } from '../menu-item-mixin.js';
 
 const tag = defineCE(
-	class extends MenuItemMixin(LitElement) {}
+	class extends MenuItemMixin(LitElement) {
+		render() { return this.text; }
+	}
 );
 
-function dispatchKeyEvent(elem, key) {
-	const eventObj = document.createEvent('Events');
-	eventObj.initEvent('keydown', true, true);
-	eventObj.keyCode = key;
-	elem.dispatchEvent(eventObj);
-}
-
 describe('MenuItemMixin', () => {
 
-	let elem;
-	beforeEach(async() => {
-		elem = await fixture(`<${tag} id="my-menu-item"></${tag}>`);
```

Comment:
I don't like this pattern -- of having a fixture that only some of the tests use.